src/app/pages/components/home/design-list.tsx

```diff
@@ -117,7 +116,6 @@ export const DesignList: FunctionComponent<DesignListProps> = (
                 columns={columns}
                 data={designs.designs}
                 expectedLength={designs.count}
-                onRowClick={(row) => setSelectedDesign(row.row.id === selectedDesign?.id ? undefined : row.row)}
```

Comment:
Why remove this?

Reply:
I guess with this change the only way to show the details panel is via the menu item?